bundles/paikkatietoikkuna/terrain-profile/instance.js

```diff
@@ -104,6 +105,7 @@ Oskari.clazz.define('Oskari.mapframework.bundle.terrain-profile.TerrainProfileBu
                 url: url,
                 success: this.showFlyout.bind(this),
                 error: function (jqXHR, textStatus, errorThrown) {
+                    me.flyout.showError();
```

Comment:
We could change the error callback to use ["fat arrow"](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions) function so that there would be no need to create a `me` variable above.